src/commands.cpp

```diff
@@ -990,10 +990,14 @@ bool CommandSetAttr::procresult(Result r)
         Node* node = client->nodebyhandle(h);
         if(node)
         {
-            if (Sync* sync = client->syncs.runningSyncByTag(tag))
-            {
-                client->app->syncupdate_remote_rename(sync, node, pa.c_str());
-            }
+            // After speculative instant completion removal, this is not needed (always sent via actionpacket code)
+            client->syncs.forEachRunningSync([&](Sync* s) {
+                    if (s->localroot->node &&
+                        node->isbelow(s->localroot->node))
```

Comment:
We might want to refactor this: forEachSyncContainingNode(node, [&](Sync *s){....}

Reply:
yes, we have three cases like that - ok, factored :-)  thanks